packages/hopli/src/identity.rs

```diff
@@ -0,0 +1,80 @@
+use crate::key_pair::create_identity;
+use clap::Parser;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use crate::utils::{Cmd, HelperErrors};
+
+/// CLI arguments for `hopli identity`
+#[derive(Parser, Default, Debug)]
+pub struct IdentityArgs {
+    #[clap(
+        help = "Password to encrypt identity files",
```

Comment:
The secret should not be passed as a parameter. Either as an environment variable or a path to a file which contains the password.

Reply:
Updated. `password` can be an env variable or pass via file, and `private_key` should be an env variable